statsmodels/tsa/statespace/_tools.pxd

```diff
@@ -29,26 +29,6 @@ cpdef int dldl(np.float64_t [::1, :] A) except *
 cpdef int cldl(np.complex64_t [::1, :] A) except *
 cpdef int zldl(np.complex128_t [::1, :] A) except *
 
-cdef int _sreorder_missing_diagonal(np.float32_t * a, int * missing, int n)
-cdef int _dreorder_missing_diagonal(np.float64_t * a, int * missing, int n)
-cdef int _creorder_missing_diagonal(np.complex64_t * a, int * missing, int n)
-cdef int _zreorder_missing_diagonal(np.complex128_t * a, int * missing, int n)
-
-cdef int _sreorder_missing_submatrix(np.float32_t * a, int * missing, int n)
-cdef int _dreorder_missing_submatrix(np.float64_t * a, int * missing, int n)
-cdef int _creorder_missing_submatrix(np.complex64_t * a, int * missing, int n)
-cdef int _zreorder_missing_submatrix(np.complex128_t * a, int * missing, int n)
-
-cdef int _sreorder_missing_rows(np.float32_t * a, int * missing, int n, int m)
-cdef int _dreorder_missing_rows(np.float64_t * a, int * missing, int n, int m)
-cdef int _creorder_missing_rows(np.complex64_t * a, int * missing, int n, int m)
-cdef int _zreorder_missing_rows(np.complex128_t * a, int * missing, int n, int m)
-
-cdef int _sreorder_missing_cols(np.float32_t * a, int * missing, int n, int m)
-cdef int _dreorder_missing_cols(np.float64_t * a, int * missing, int n, int m)
-cdef int _creorder_missing_cols(np.complex64_t * a, int * missing, int n, int m)
-cdef int _zreorder_missing_cols(np.complex128_t * a, int * missing, int n, int m)
-
```

Comment:
Not related to the fused types, just makes it easier to reason about code when these aren't exposed.